src/container/ContainerBase/index.ts

```diff
@@ -1,12 +1,17 @@
+export const enum IteratorType {
+  /* eslint-disable no-unused-vars */
```

Comment:
Eslint v8.22.0 works fine to me.

Reply:
I tried updating eslint, but I keep getting problems.

Reply:
Try to change rule name from `no-unused-vars` to `@typescript-eslint/no-unused-vars`?

Reply:
It works! I'll commit to change the eslint setting